examples/python-poetry/devenv.yaml

```diff
@@ -1,3 +1,5 @@
 inputs:
   nixpkgs:
     url: github:NixOS/nixpkgs/nixpkgs-unstable
+  devenv:
```

Comment:
I don't think this is needed, because it's added by the testing script to point to the local branch (it also fails CI as it now has double devenv)